apps/site/public/glimm.js

```
@@ -0,0 +1,1829 @@
+    gl.uniform1f(uIdea3Refraction, state.ideas.refraction);
+    gl.uniform1f(uIdea4Chromatic, state.ideas.chromaticDispersion);
+    gl.uniform1f(uIdea5Noise, state.ideas.noiseEdge);
+    gl.uniform1f(uIdea6Camera, state.ideas.cameraSweep);
+    gl.uniform1f(uIdea7Bloom, state.ideas.bloom);
+    gl.uniform1f(uIdea8Sparkles, state.ideas.sparkles);
+    gl.uniform1f(uIdea9Curl, state.ideas.curlWake);
+    if (state.useTexture > 0 && state.textureSource) {
+      ensureTexture();
+      gl.activeTexture(gl.TEXTURE0);
+      gl.bindTexture(gl.TEXTURE_2D, tex);
+      try {
```

ESLint

[error] Insert `······`

(prettier/prettier)